assets/src/scss/components/home/_get-in-touch.scss

```diff
@@ -0,0 +1,226 @@
+main {
+
+    // mobile view
+    .message-area-container {
+
+
+        @media screen and (min-width: $tablet) {
+            display: flex;
+            padding-right: 15px;
+            padding-left: 15px;
+        }
+
+        @media screen and (min-width: $desktop) {
+            padding-left: 7.8125rem;
+            padding-right: 7.8125rem;
+            padding-bottom: 80px;
+        }
+    }
+
+    .message-area {
+        padding-left: 15px;
+        padding-right: 15px;
+
+        @media screen and (min-width: $tablet) {
+            width: 65%;
+        }        
+    }
+
+
+    .message-area h5 {
+        margin-bottom: 8px;
+        font-size: 20px;
+        color: #212121;
+        font-weight: 700;
+        font-family: inherit;
+        text-transform: uppercase;
+
+        @media screen and (min-width: $desktop) {
+            margin-bottom: 8px;
+            font-size: 26px;
+            line-height: 24px;
+        }
+
+    }
+
+    .update-contact-form {
+        position: relative;
+        margin-bottom: 50px;
+        background: #fafafa;
+        padding: 40px;
+    }
+
+
+    .action-page {
+        
+        @media screen and (min-width: $desktop) {
```

Comment:
Can we add some space for readability like so :
` @media screen and ( min-width: $desktop ) {`